_posts/blog/2019-12-16-accumulo-proxy.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Accumulo clients in other languages
+---
+
+Apache Accumulo has an [Accumulo Proxy] that allows communication with Accumulo using clients written
+in languages other than Java. This blog post shows how to run the Accumulo Proxy process using [Uno]
+and communicate with Accumulo using a Python client.
+
+First, clone the [Accumulo Proxy] repository.
+
+```bash
+git clone https://github.com/apache/accumulo-proxy
+```
+
+Assuming you have [Uno] set up on your machine, configure `uno.conf` to start the [Accumulo Proxy]
+by setting the configuration below:
+
+```
+export POST_RUN_PLUGINS="accumulo-proxy"
+export PROXY_REPO=/path/to/accumulo-proxy
+```
+
+Run the following command to set up Accumulo again. The Proxy will be started after Accumulo runs.
+
+```
+uno setup accumulo
+```
+
+After Accumulo is set up, you should see the following output from uno:
+
+```
+Executing post run plugin: accumulo-proxy
+Installing Accumulo Proxy at /path/to/fluo-uno/install/accumulo-proxy-2.0.0-SNAPSHOT
+Accumulo Proxy 2.0.0-SNAPSHOT is running
+    * view logs at /path/to/fluo-uno/install/logs/accumulo-proxy/
+```
+
+Next, follow the instructions below to create a Python client that creates an Accumulo table
+named `pythontest` and writes data to it:
```

Comment:
Could mention which python version is required for using python.

Reply:
I added the version. I also created an issue to have it work with Python 3.0. https://github.com/apache/accumulo-proxy/issues/13